test/collateral/extensions/oni-language-features/extension.js

```diff
@@ -62,9 +62,9 @@ function activate(context) {
 
 	const signatureHelpProvider = {
 		provideSignatureHelp: (_document, _position, _token, _context) => {
-			const signature1 = new vscode.SignatureInformation("signature 1", null);
+			const signature1 = new vscode.SignatureInformation("signature 1", "signature 1 documentation");
 			signature1.parameters = [
-				new vscode.ParameterInformation("parameter 1", null)
+				new vscode.ParameterInformation("parameter 1", "parameter 1 documentation")
 			];
```

Comment:
@zbaylin - if you decide you want to test signature help via the `oni-dev-extension`, this will be a good sample to start with 👍